tests/run_tests.py

```diff
@@ -163,18 +163,19 @@ def _finalize_coverage(cov):
         print("Note: Full html coverage report can be found in the coverage_html_report folder.")
 
 
-def _initialize_logging(log_to_console):
+def _initialize_logging():
     """
     Sets up a log file for the unit tests and optionally logs everything to the console.
-
-    :param log_to_console: If True, all Toolkit logging will go to the console.
     """
     import tank
     tank.LogManager().initialize_base_file_handler("run_tests")
 
     if options.log_to_console:
         tank.LogManager().initialize_custom_handler()
 
+    if options.debug:
```

Comment:
Ah. I personally do `TK_DEBUG=1 ./run_tests.sh` when I want to enable debug logging. To each their own! :)

Reply:
ah, didn't think about that! i thought it's nice to have the same thing we have in the tank command for consistency!